supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -92,11 +92,16 @@ case class Run(runId: String, genomicsInterface: Genomics) {
   def status(): RunStatus = {
     val op = genomicsInterface.operations().get(runId).execute
     if (op.getDone) {
-      // If there's an error, generate a Failed status. Otherwise, we were successful!
       val eventList = getEventList(op)
+      val ceInfo = op.getMetadata.get ("runtimeMetadata").asInstanceOf[GArrayMap[String,Object]].get("computeEngine").asInstanceOf[GArrayMap[String, String]]
```

Comment:
FWIW this line bothered me due to the runtime danger, but this whole function is wrapped in a Try so I went back to not worrying